03_launch_mgmt_cluster.sh

```diff
@@ -249,16 +264,20 @@ function create_clouds_yaml() {
 }
 
 create_clouds_yaml
-init_minikube
-sudo su -l -c 'minikube start' "${USER}"
-if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
-  sudo su -l -c 'minikube ssh "sudo ip -6 addr add '"$CLUSTER_PROVISIONING_IP/$PROVISIONING_CIDR"' dev eth2"' "${USER}"
-else
-	sudo su -l -c "minikube ssh sudo brctl addbr $CLUSTER_PROVISIONING_INTERFACE" "${USER}"
-	sudo su -l -c "minikube ssh sudo ip link set $CLUSTER_PROVISIONING_INTERFACE up" "${USER}"
-	sudo su -l -c "minikube ssh sudo brctl addif $CLUSTER_PROVISIONING_INTERFACE eth2" "${USER}"
-	sudo su -l -c "minikube ssh sudo ip addr add $INITIAL_IRONICBRIDGE_IP/$PROVISIONING_CIDR dev $CLUSTER_PROVISIONING_INTERFACE" "${USER}"
 
+if "${EPHEMERAL_CLUSTER}" == "kind"; then
+  kind create cluster
```

Comment:
To start kind cluster with specific version would be better. kind create cluster --image=kindest/node:v1.16.3
This should use the KUBERNETES_VERSION from li

Reply:
Fixed